ldap/servers/slapd/result.c

```diff
@@ -1999,6 +1999,64 @@ notes2str(unsigned int notes, char *buf, size_t buflen)
     return (buf);
 }
 
+static void
+log_op_stat(Slapi_PBlock *pb)
+{
+
+    Connection *conn = NULL;
+    Operation *op = NULL;
+    Op_stat *op_stat;
+    struct timespec duration;
+    char stat_etime[ETIME_BUFSIZ] = {0};
+
+    slapi_pblock_get(pb, SLAPI_CONNECTION, &conn);
+    slapi_pblock_get(pb, SLAPI_OPERATION, &op);
+    op_stat = op_stat_get_operation_extension(pb);
+
+    if (conn == NULL || op == NULL || op_stat == NULL) {
+        return;
+    }
+    /* process the operation */
+    switch (op->o_tag) {
+        case LDAP_REQ_BIND:
+        case LDAP_REQ_UNBIND:
+        case LDAP_REQ_ADD:
+        case LDAP_REQ_DELETE:
+        case LDAP_REQ_MODRDN:
+        case LDAP_REQ_MODIFY:
+        case LDAP_REQ_COMPARE:
+            break;
+        case LDAP_REQ_SEARCH:
+            if ((LDAP_STAT_READ_INDEX & config_get_statlog_level()) &&
+                op_stat->search_stat) {
+                struct component_keys_lookup *key_info;
+                for (key_info = op_stat->search_stat->keys_lookup; key_info; key_info = key_info->next) {
+                    slapi_log_stat(LDAP_STAT_READ_INDEX,
```

Comment:
slapi_log_stat proto is missing in include file

Reply:
done

Reply:
Weird there is still a warning ...